### ISSUE TITLE
Error building kotlin-native/samples/calculator` project : llvmstubs not found 

### ISSUE THREAD

**Issue by joao-parana** [JetBrains/kotlin-native#1740]
I get this error bellow when try tu build `kotlin-native/samples/calculator` project. 

How can I set the `java.library.path` to correct this ? 

I'm using - macOS 10.13.5 and v0.7.1 kotlin-native Release.

```
Extracting dependency: /Users/admin/.konan/cache/clang-llvm-5.0.0-darwin-macos.tar.gz into /Users/admin/.konan/dependencies
error: compilation failed: no llvmstubs in java.library.path

 * Source files: Parser.kt
 * Compiler version info: Konan: 0.7.1-dev / Kotlin: 1.2.60
 * Output kind: FRAMEWORK

exception: java.lang.UnsatisfiedLinkError: no llvmstubs in java.library.path
```

**Comment by olonho** [comment]
Hmm, looks suspicious, do you use bash as your shell?
Variable is set in https://github.com/JetBrains/kotlin-native/blob/986f5220dfe473feb4cd4f0c03cf26d4863435a8/cmd/run_konan#L69.

**Comment by olonho** [comment]
Can you build Kotlin/Native from sources per README.md in this repo?

**Comment by olonho** [comment]
No updates, closing

**Comment by KevinOfNeu** [comment]
@olonho I run into 'no llvmstubs in java.library.path' problems too when i wanna  run K2nativeKt in IDEA, I fixed this issue by adding '-Djava.library.path=path/dist/konan/nativelib' to VM options.

**Comment by soywiz** [comment]
Reproduced a similar problem on linux here:
https://travis-ci.org/korlibs/klock/jobs/549794325#L3151

```
Downloading native dependencies (LLVM, sysroot etc). This is a one-time action performed only on the first run of the compiler.
Downloading dependency: https://download.jetbrains.com/kotlin/native/clang-llvm-6.0.1-linux-x86-64.tar.gz (509.0 MiB/509.0 MiB). Done.
Extracting dependency: /home/travis/.konan/cache/clang-llvm-6.0.1-linux-x86-64.tar.gz into /home/travis/.konan/dependencies
Downloading dependency: https://download.jetbrains.com/kotlin/native/target-gcc-toolchain-3-linux-x86-64.tar.gz (58.4 MiB/58.4 MiB). Done.
Extracting dependency: /home/travis/.konan/cache/target-gcc-toolchain-3-linux-x86-64.tar.gz into /home/travis/.konan/dependencies
Downloading dependency: https://download.jetbrains.com/kotlin/native/libffi-3.2.1-2-linux-x86-64.tar.gz (55.1 kiB/55.1 kiB). Done.
Extracting dependency: /home/travis/.konan/cache/libffi-3.2.1-2-linux-x86-64.tar.gz into /home/travis/.konan/dependencies
e: Compilation failed: /home/travis/.konan/kotlin-native-linux-1.3/konan/nativelib/libllvmstubs.so: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.17' not found (required by /home/travis/.konan/kotlin-native-linux-1.3/konan/nativelib/libllvmstubs.so)
 * Source files: DateException.kt, DateFormat.kt, DateTime.kt, DateTimeRange.kt, DateTimeSpan.kt, DateTimeSpanFormat.kt, DateTimeTz.kt, DayOfWeek.kt, ISO8601.kt, KlockLocale.kt, Measure.kt, Month.kt, MonthSpan.kt, PatternDateFormat.kt, PerformanceCounter.kt, TimeFormat.kt, TimeProvider.kt, TimeSpan.kt, TimezoneOffset.kt, Year.kt, YearMonth.kt, FastForEach.kt, KlockInternal.kt, MicroStrReader.kt, NiceStr.kt, internal.kt, KlockInternal.kt
 * Compiler version info: Konan: 1.3 / Kotlin: 1.3.40
 * Output kind: LIBRARY
e: java.lang.UnsatisfiedLinkError: /home/travis/.konan/kotlin-native-linux-1.3/konan/nativelib/libllvmstubs.so: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: version `GLIBCXX_3.4.17' not found (required by /home/travis/.konan/kotlin-native-linux-1.3/konan/nativelib/libllvmstubs.so)
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1857)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at llvm.llvm.<clinit>(llvm.kt:4383)
	at org.jetbrains.kotlin.backend.konan.llvm.Int64.<init>(LlvmUtils.kt:90)
	at org.jetbrains.kotlin.backend.konan.llvm.LocalHash.<init>(HashUtils.kt:53)
	at org.jetbrains.kotlin.backend.konan.llvm.ContextUtilsKt.getLocalHash(ContextUtils.kt:246)
...
```

**Comment by artdfel** [comment]
Hello @soywiz!
Can you tell me, what version of `libstdc++` do you have?

**Comment by soywiz** [comment]
I found the issue yesterday. I will provide the context and the solution later today just in the case someone else stump into this

**Comment by soywiz** [comment]
If this happens to someone else. It seems that travis for android is using Ubuntu 12.04. A extremely old ubuntu version. The libstdc++ included there supports up to 3.4.16.
After some trials and errors I did a few changes here: https://github.com/korlibs/klock/pull/52
